### ISSUE TITLE
Height calculation of columns to equalize

### ISSUE THREAD

**Issue by csanyiarpad** [jsliang/eqHeight.coffee#16]
I have columns having a 10px padding which I would like to equalize.
The tallest column's text will always overlap (the padding or even the border), because the `height()` function does not count with paddings and borders. Therefore I suggest using the `outerHeight()` function, which solves the mentioned problem.

``` coffeescript
# Get max height of marked_columns
    max_col_height = 0
    marked_columns.each () ->
        max_col_height = Math.max($(this).outerHeight(), max_col_height)
```

**Comment by jsliang** [comment]
Hi @csanyiarpad,
Yes, I agree that outerHeight() makes more sense. Would you like to send a pull request for this?

**Comment by jsliang** [comment]
Closed by #17.
